plots/electricity_bills.py

```diff
@@ -0,0 +1,255 @@
+    lv_load = n.loads_t.p_set[lv_buses]
+    
+    # electricity consumption of residential heat techs in low_voltage bus of links
+    rh_elec_links = n.links.query("carrier in @rh_techs_elec").index
+    rh_techs_mapping = n.links.loc[rh_elec_links, "bus0"].to_dict()
+    rh_techs_consume = n.links_t.p0[rh_elec_links]
+    rh_techs_consume = rh_techs_consume.rename(columns=rh_techs_mapping)
+    rh_techs_consume = rh_techs_consume.groupby(level=0, axis=1).sum()
+    
+    # electricity consumption of land transport EV in low_voltage bus in links
+    ev_charge_links = n.links.query("carrier in @ev_tech_charge").index
+    ev_charge_mapping = n.links.loc[ev_charge_links, "bus0"].to_dict()
+    ev_tech_consume = n.links_t.p0[ev_charge_links]
+    ev_tech_consume = ev_tech_consume.rename(columns=ev_charge_mapping)
+    ev_tech_consume = ev_tech_consume.groupby(level=0, axis=1).sum()
+    
+    # electricity prosumption of land transport EV to low_voltage bus in links
+    ev_discharge_links = n.links.query("carrier in @ev_tech_discharge").index
+    ev_discharge_mapping = n.links.loc[ev_discharge_links, "bus1"].to_dict()
+    ev_tech_prosume = n.links_t.p1[ev_discharge_links]
+    ev_tech_prosume = ev_tech_prosume.rename(columns=ev_discharge_mapping)
+    ev_tech_prosume = ev_tech_prosume.groupby(level=0, axis=1).sum()
+    
+    # electricity prosumption of micro CHP to low_voltage bus in links
+    rh_mCHP_links = n.links.query("carrier in @rh_techs_mCHP").index
+    rh_mCHP_lv_mapping = n.links.loc[rh_mCHP_links, "bus1"].to_dict()
+    rh_mCHP_prosume = n.links_t.p1[rh_mCHP_links]
+    rh_mCHP_prosume = rh_mCHP_prosume.rename(columns=rh_mCHP_lv_mapping)
+    rh_mCHP_prosume = rh_mCHP_prosume.groupby(level=0, axis=1).sum()
+    
+    # gas consumption of micro CHP in EU_gas bus in links
+    rh_mCHP_gas_mapping = n.links.loc[rh_mCHP_links, "bus0"].to_dict()
+    rh_mCHP_consume = n.links_t.p0[rh_mCHP_links]
+    rh_mCHP_consume = rh_mCHP_consume.rename(columns=rh_mCHP_gas_mapping)
+    rh_mCHP_consume = rh_mCHP_consume.groupby(level=0, axis=1).sum()
+    
+    # gas consumption of gas boilers in EU_gas bus in links
+    rh_gas_links = n.links.query("carrier in @rh_techs_gas").index
+    if not rh_gas_links.empty:   
+        rh_gas_mapping = n.links.loc[rh_gas_links, "bus0"].to_dict()
+        rh_gas_consume = n.links_t.p0[rh_gas_links]
+        rh_gas_consume = rh_gas_consume.rename(columns=rh_gas_mapping)
+        rh_gas_consume = rh_gas_consume.groupby(level=0, axis=1).sum()
+    else:
+        rh_gas_consume = pd.DataFrame()
+    
+    # total electricity consumption in low_voltage nodes
+    total_elec_load = (lv_load + rh_techs_consume + ev_tech_consume + ev_tech_prosume + rh_mCHP_prosume).multiply(n.snapshot_weightings.stores, axis=0)
+    
+    # total gas consumption in EU_gas node
+    if not rh_gas_links.empty: 
+        total_gas_load = (rh_mCHP_consume + rh_gas_consume).multiply(n.snapshot_weightings.stores, axis=0)
```

Comment:
Probably prosumer part should be included here
```suggestion
        total_gas_load = (rh_mCHP_consume +  rh_mCHP_prosume + rh_gas_consume).multiply(n.snapshot_weightings.stores, axis=0)
```

Reply:
> Probably prosumer part should be included here

The gas load will not have `rh_mCHP_prosume`, because `rh_mCHP_prosume` refers to electricity production in bus1. We will have only `rh_mCHP_consume` (refers to bus0 of link) in gas load.